java/ql/src/semmle/code/java/dataflow/FlowSummary.qll

```diff
@@ -8,7 +8,9 @@ private import internal.DataFlowDispatch
 private import internal.DataFlowPrivate
 
 // import all instances below
-private module Summaries { }
+private module Summaries {
```

Comment:
Elaborate the comment a bit -- why are we importing them? Presumably there's some particular abstract class definition we're collecting implementations of?

Reply:
The abstract class is primarily `SummarizedCallable` (there's also `RequiredSummaryComponentStack`) - and extending that is the main reason to import `FlowSummary.qll`.
Perhaps extend the comment like this?
```
// import all instances of SummarizedCallable below
```